### ISSUE TITLE
How to train the whole model ? 

### ISSUE THREAD

**Issue by lubovbyc** [Blade6570/icface#7]
Hi,
    We tried to reproduce your approach by re-training your model. However, details of the training procedure are not mentioned in the paper. Currently we struggle with the issue that both G_N and G_A tend to ignore the embedded face attributes FA and always generate the same pose/expression as the source image.
    Can you share some insights about this issue or the training strategy? 
Thanks

**Comment by Blade6570** [comment]
Hi, the following part is less emphasized in the paper but is very important to make the training stable:  

>  we stabilize the training of the GN by using generated images from GA with neutral attributes as a pseudo ground truth.

Make sure that pseudo Ground truths are detached in pytorch. 
> we have also observed that you can increase the regularization for facial attribute loss. 
> We have also observed that replacing the identity classification part with a pretrained facial recognition network also improves the performance. Use this pretarined network for perceptual loss.
